engine/engine_test.go

```diff
@@ -1500,6 +1500,24 @@ func TestQueriesAgainstOldEngine(t *testing.T) {
 			end:   time.Unix(3000, 0),
 			step:  2 * time.Second,
 		},
+		{
+			name: "absent with present series",
+			load: `load 30s
+				absent{pod="nginx-1", series="1"} 0
+				absent{pod="nginx-2", series="1"} 0`,
+			query: "absent(metric)",
```

Comment:
This test case seems wrong itself. Your metric name is `absent` but you are calling `absent` for `metric`. `metric` is not present in your test data.